plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/properties/ObjectPropertyDescriptor.java

```diff
@@ -280,7 +283,10 @@ public boolean hasFeature(@NotNull String feature) {
             case DBConstants.PROP_FEATURE_PASSWORD:
                 return this.isPassword();
         }
-        return false;
+
+        String[] features = propInfo.features();
```

Comment:
Again, this function may be called 10000 times per second. Do not use expensive operations including streams.
Use check for empty array (99% cases) + ArrayUtils.contains.